index.js

```diff
@@ -1,10 +1,11 @@
-﻿﻿const nibSpecificCountries = require("./countries/nib");
+﻿const nibSpecificCountries = require("./countries/nib");
 const cgSpecificCountries = require("./countries/coverGenius");
 const universalCountries = require("./countries/universal");
 
-const countries = universalCountries
-  .concat(nibSpecificCountries)
-  .concat(cgSpecificCountries);
+// The order matters here - be careful
```

Comment:
Not sure why the order matters but this sounds like something it might be useful to have a test for. Then the developer doesn't have to remember to be extra careful :)

Reply:
But I agree that we should have a test case.